hepcrawl/parsers/jats.py

```diff
@@ -314,10 +314,25 @@ def get_affiliation(self, id_):
                 no match.
         """
         affiliation_node = self.root.xpath('//aff[@id=$id_]', id_=id_)[0]
-        affiliation = remove_tags(affiliation_node, strip='self::label').strip()
+        affiliation = remove_tags(
+            affiliation_node,
+            strip='self::label | self::email'
+        ).strip()
 
         return affiliation
 
+    def get_emails_from_refs(self, id_):
+        """Get the emails from the node with the specified id.
+
+        Args:
+            id_(str): the value of the ``id`` attribute of the node.
+
+        Returns:
+            List[str]: the emails from the node with that id or [] if none found.
+        """
+        email_nodes = self.root.xpath('//aff[@id=$id_]/email', id_=id_)
+        return [node.root.text for node in email_nodes]
```

Comment:
you shouldn't use `Selector.root` usually, as that involves dealing with the underlying `Element` node, which is wrapped by `Selector`. Instead, you can do something like
```
emails = self.root.xpath('//aff[@id=$id_]/email/text()', id_=id_).extract()
return emails
```
Be aware of https://doc.scrapy.org/en/latest/topics/selectors.html#using-text-nodes-in-a-condition though and the difference between `string(selector)` and `selector/text()` but here it should be fine.